bin/expose-host-agent

```diff
@@ -13,7 +13,7 @@ fi
 
 # Bind the SSH agent unix socket to a TCP socket on the loopback interface
 # This allows processes inside a Docker build to use keys from the agent
-/usr/local/bin/socat tcp4-listen:24249,bind=127.0.0.1,reuseaddr,fork unix:${SSH_AUTH_SOCK} &
+socat tcp4-listen:24249,bind=127.0.0.1,reuseaddr,fork unix:${SSH_AUTH_SOCK} &
```

Comment:
we could use `"$(brew --prefix)/bin/socat"` but in the other script we use socat directly see https://github.com/gocardless/expose-host-agent/blob/master/bin/with-host-agent#L15